app/controllers/concerns/eventable.rb

```diff
@@ -0,0 +1,30 @@
+module Eventable
```

Comment:
possibly this should be named `MoveEventable` or `Moves::Eventable` as its focus is purely on `move` events. Coming down the line are `journey` events, `people` events (`allocation` events??).

Reply:
Cool I'll try that 👍